examples/DRISE.ipynb

```diff
@@ -0,0 +1,523 @@
f/ojt6+u0crx9DgxT5Hf/I8/cPvmhn7YkcuJmFYAraQwfPHll7x6FZimifuHRyG97HeYRiU5HY8Cv66STuGMJVfFHCIoJ6M1hIQjIAUrQHHXEWu8+Dud6pjPC0+nka73+GFDLAunqXCeIufjRx7uH7l58V/59V//W9588XnLnnP4vufm5ob5PHI+nwghoVXFW1Egz0u8nB2SJuFa12cYhoFKJYSpdarSCZ3PZ2Gcdu7iL13zAi878JYwYd1q+6GtGdxl16eRwrXvOyEDyQF1CYzMpZBTout7cozE9uu6WTG8k251GkdiiE2mr4k5SvCHFtBCLZKyId1WQ0TlfBGJyVXx580K//yOS3tyhv32Bcbd0F29BdWBtqQw40rEZIle164X1Vo1MkpRGmqEWqhpQaWZUjIxRLp+1y64Jq08H/CbLSUukjHle8J5oszizA5LwFhPxsmHWxsxOFehb5vmFrctKDFFGQtq5wihRVJ7wTutGnOtDdaaZ6x+LRfRh1YykhDVS6EfOlKSqiOXDDSemBGYsHipcvtnLjsrEWKIGGKtQMS78YxWkbZfmITWPUcCrBfxmloqYpL2gCnQVos83wpxJCUZ8fi6Cj+E7O60JTcVY4yJmEuLPpGdl3cOlGmPlGKz6en6vajdsmSsLctM1/XkDMNmSy1wd/9IKRNhWghRRqlPhyectYQl8PLlNZvBsN9vUMrwdJiZxpHTwbPpDVYVVPHsNlsOh3PL8rFY15Fi4nQ4UlOgt5rOaVH3xdI6XMM8zcQYWCPPFRla10/N9B6uNo7OVpzKqMawTElGy8YaVHs/SnseqrLMIbEsQWwS2soFlQqbzhHnE73vxKO2RJkoFIFKh1AYp4VcM94Z4iLjIVOqVNBVvDU5JQyasERiWsgyaxOI8XbANr/g8XhGV7BWE8LMppdYGNf4cN2wwXqPIqN1bjElEecMNhm22y2V0vLCOrAOsqIkmKdFvH5K4V3HNC7M09zk0JbbF9d0vWUJMzF2/O73X3O9kXDBcZl5/fYz/vjtN4Rc+eWvunbRwd3HB4ZNR99bxkW+l/3tS0z3Lf1mwzQtfHh3zxLhiy++5HjKvHv3js1mj7aR3GLlpylzPIzMLdR0u9tye3st+5GmoAs5s9vt6Xcbam4ElpCEAG+HlmTcUs5bF+AbhUVbh9a27bIVxjmMtijtqVWWBcZ5nO45T4nDeGYMC27TM6XEkivbwUGozDnxNH3gw90DT48PvP38c7782U/58qufsSyBYdjy+tUbThUO+URu3U8MWSgvekW/5VbY2svKwFrTzgoxBgvJphWi6wTGGEpqQaaqScuzpDpro5qoram/TStm1SeghJIvsSo5ZVENNhCBUoqQkyDwdCefLbhMnlKMuLZ6McbKzrO2JJB2GUtX1kQnckM2jcBzWO76dX+0i0vZDcOwpd+9odChhyuKMuRc8MbAckClE5hb8bI0j0VVYiyWbUCllIgqCWV7XH8NWqN9gVIJpwllHBSFdgOozHg6cPj2N/zx9/+Fx4d35BqpVtzwpWohvdciickX93YTI7QKpJTCGk+mWndIuxRWlqH7xMgri/1MzgljtYy2mgJNaBwiwy+1SUCV8PRYtYbqOaJaaUOtGY1B05JAi0jwaxtryuUpZtO+7wgxEpblQoRed2DCCeuafB5Ww/GFf5ZFlLF2jaIMgpTEtJoXmWX3vcQNyOtBg3UuhBQY+i3WegbvmZfINI9YZ9hfbci1srPXpCSg3Glc0Ebz5rNXaPvIEgLGQAiJ2mbVXec5HI6UXLG259XLF+y2Ms8fTwem08jt9Q5jnKg3W/BojFk4gLnS9xviJBlhtfnRnPekEJimGaMFTJqzjBxqycRlJoWMKolh8AyDofcaZxUQLz4Xpa3ErIDs8ADrNMbKbrKq3HiQmVwWQlgwrsMohdGyDytFM84JSsVoIXa4bsOmt1hT8YBXCmpGOU1GsFYaDU31mhFq+DiO6BCIyyzUeW1lca2EF2q7nqEf2PaOHKXLiHEi5wXnZV/XdR1hnITBqCsxL1gMu92GsCws00xcIrXI1yxaDmhhWGZUlc5dOXM58LfbDRjHeZrY9gNhHNlsB7TR/M3f/A2lqMvnLS6RmCL73Ybz+cybt28pJXN8OnDz8iUf7x74/Z++4e7hgLNbfv9Pf5BIexTRa54OJ/quI6SA9YqilJD9k2DUtkOPdpZu6HGdB60YxwlQ5GURn6bikijQ9QPXqiWWq9rUhlIsP/sooWqFsU7EOEVR0RL6qD1LrCSgGzb0LeizNFxbqoXS0qWDKsSwsEx/5PD0yA/vf+C//P1/xRoJW+2c/K2ZKTlSi4wJ9SpJ16LQE9apmN9z+xzLBSAouhQTzgler9YsXqwksPAUBXxgrMNqSVM3iIVCG9nfhmWh871E++SMMQMFKZpX8LcwaaXAX0UZcl60aVTDN+UiHFYJYfVy+dbSxEXy+1RLZPbey/Oe8kWIUj45iy/dxI91cV3v3+I3L6h6QHU9xVkyYrZVpbn3D++o9w/Y4QZ/dQvdIL4TKjUuUAppCbhuQHcDVSnKfELrSp5FiumGvdzGaWE5fOTw/hu++cPf88O333J4jCzREXOWoEKjiAlQ4rlZQx4lPkBf5JZaa0ncrOXif8rN+2AkrpeQA/1mkC5odbOjMNWgmkFYGUVsF0Up9eJ9UEohuXXywahJRpDWChqmlopRhhwr2ni860hJ0CpoMTXP89TUjBGUwn3C7FJaVDirwrAUuaTWMVfU4WIB0E2tuYI0oV6AmcYO+K4jNvWQt4ZpiZAlBbYAUSlyrCSKfK/W0fdD88pBqRJJv2ihcy8x8ubNK774vMdZy939I09PJ5ZFRgLzUvH9NR8+zHh34uVLxfX1QN/1PN7DMs0ctcIYSGnmcJ7ofMdQNtQqF1TvNAuRMD4RSmzg0QZFLrILuggzSiGHGUrAGY3ZiIzfeo0yki4NDm2laxAmoiCrtO6piApVK+Ha1RxJqilIq0WpfImN
```

Comment:
D-RISE requires the class probabilities, not just the class label as returned by torchvision detection models. This might require some additional model surgery to get the logits (vs. just the class label).